python/tests/unit/arcticdb/version_store/test_append.py

```diff
@@ -288,3 +289,100 @@ def test_(initial: InputFactories, append: InputFactories, match, swap, lmdb_ver
     to_append, _ = append.make(abs(next_start), 1)
     with pytest.raises(NormalizationException):
         lib.append("s", to_append, match=match)
+
+
+def test_append_non_sorted_exception(lmdb_version_store):
```

Comment:
I think we also need a test that writes some `UNSORTED` data and then calls `append` with `ensure_sorted=True`, as this should fail.

Reply:
Idem to previous comment

Reply:
Is this identical to the comment about update? Update is illogical. Append isn't.